applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -45,7 +45,8 @@ config BT_AUDIO_PRESENTATION_DELAY_US
 config BT_AUDIO_MAX_TRANSPORT_LATENCY_MS
 	int "Max transport latency"
 	range 5 4000
-	default 20
+	default 20 if TRANSPORT_CIS && BT_LL_ACS_NRF53
+	default 10
```

Comment:
Would this not restrict use of PTO and FT? 20 ms latency would allow a PTO=1 or FT=2.

Reply:
Not sure I follow. Can you elaborate? This is set as TRANSPORT_CIS on BT_LL_ACS_NRF53 does not work well on 10 ms.